### DISCUSSION TITLE
Use a universal GUI frontend for multi-platform maybe easier to maintain an update ?

### DISCUSSION THREAD

**Discussion by q2333gh** [spacedriveapp/spacedrive#1713] [General]
Such as electron? Flutter ?

**Comment by HeavenVolkoff** [comment]
Hello, we are already building Spacedrive with an universal GUI that support multiple platforms (Windows, macOS and Linux). It is called Tauri, you can learn more about it on it's website: https://tauri.app/ or GitHub repo: https://github.com/tauri-apps/tauri
